curriculum/challenges/english/16-the-odin-project/top-learn-data-types-and-conditionals/learn-data-types-and-conditionals-question-b.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 65e97164484dd50f720e6fe7
+title: Learn Data Types and Conditionals Question B
+challengeType: 15
+dashedName: learn-data-types-and-conditionals-question-b
+---
+# --description--
+
+Now that you have gone through the points listed in the assignment of the previous challenge, you should have a good understanding of how strings can be manipulated in JavaScript.
+
+Currently, this given string has a trailing space at the end after the exclamation mark. To solve this, you need to remove the trailing space from the string using a predefined method.
+
+```javascript
+let greeting = "Hello World!  ";
+
+console.log(greeting);
+```
+
+<a href="https://www.freecodecamp.org/news/javascript-string-handbook" target="_blank"> Link to Guide </a>
+
+# --question--
+
+## --text--
+
+Following the problem above, which method should be used to remove the trailing space from the string?
```

Comment:
I feel like this question could be misleading because technically `trim()` and `trimEnd` would remove space from the end of the string.

<img width="339" alt="Screenshot 2024-03-12 at 9 26 10 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/6deb2d6a-6b72-4aab-a3e0-2ed5ac23d9c8">

Maybe the question, could be reworded to say which method is used to specifically remove space only from the end?